api/v1/models/MailingList.js

```diff
@@ -0,0 +1,27 @@
+var Model = require('./Model');
+var User = require('./User');
+var MailingListUser = require('./MailingListUser');
+
+var MailingList = Model.extend({
+	tableName: 'mailing_lists',
+	idAttribute: 'id'
+});
+
```

Comment:
Also is there going to be a programatic way of adding a new list (I saw the list of lists and the SQL)?